projects/addon-editor/components/toolbar/toolbar-navigation-manager.directive.ts

```diff
@@ -50,7 +50,7 @@ export class TuiToolbarNavigationManagerDirective {
         for (const el of tools) {
             const focusableElement = tuiIsNativeMouseFocusable(el)
                 ? el
-                : tuiGetClosestFocusable(el, false, el, false);
+                : tuiGetClosestFocusable(el, el, false, false);
```

Comment:
It can be a good idea to change function signature from

```ts
function tuiGetClosestFocusable(
    initial: Element,
    root: Node,
    prev: boolean = false,
    keyboard: boolean = true,
): HTMLElement | null {}
```

to 

```
function tuiGetClosestFocusable({
    initial,
    root,
    prev = false,
    keyboard = true,
}: {
    initial: Element;
    root: Node;
    prev: boolean;
    keyboard: boolean;
}): HTMLElement | null {}
```

Could you add TODO (for v3.x) comment, please ?

Reply:
https://github.com/Tinkoff/taiga-ui/issues/2390